### DISCUSSION TITLE
FAIRification of the DFC Standard

### DISCUSSION THREAD

**Discussion by RaggedStaff** [datafoodconsortium/standarddocumentation#9] [Ideas]
I'm curious if it has been discussed previously, whether we could align with the [FAIR principles](https://www.go-fair.org/fair-principles/l) for Data Management & Stewardship ? 

I think the standard is mostly well-aligned already but we probably specific concerns to address around [principle R1 (Reuse)](https://www.go-fair.org/fair-principles/r1-metadata-richly-described-plurality-accurate-relevant-attributes/).

What do you think @lecoqlibre @simonLouvet @RachL  ?

**Comment by simonLouvet** [comment]
I know FAIR concept.
I need examples of what you think we could do better.
